pyproject.toml

```diff
@@ -30,7 +30,7 @@ dependencies = [
     "grpcio-status>=1.26.0",
     "nltk>=3.9.1",
     "numpy>=1.14.0,<3.0.0",
-    "pandas>=1.1.0,<2.3",
+    "pandas>=1.1.0,<2.4",
```

Comment:
@prmukherj @hpohekar @mkundu1 

Do we really need to maintain a rolling 2.x limit for pandas?

Reply:
@seanpearsonuk I think we can change it to "pandas>=1.1.0"

Reply:
@prmukherj @hpohekar @mkundu1 I was thinking we could still have an upper limit, but something broad like `<3` to guard against potential breaking changes in a future major version. But I am still open to keeping the present policy in place if it is well-founded.